tests/e2e/kine/kine_test.go

```diff
@@ -161,7 +161,7 @@ var _ = Describe("Verify Basic Cluster Creation with Kine", Ordered, func() {
 	It("Verifies dns access", func() {
 		_, err := tc.DeployWorkload("dnsutils.yaml")
 		Expect(err).NotTo(HaveOccurred())
-		cmd := "kubectl --kubeconfig=" + tc.KubeconfigFile + " exec -i -t dnsutils -- nslookup kubernetes.default"
+		cmd := "kubectl --kubeconfig=" + tc.KubeconfigFile + " exec -i -t dnsutils -- nslookup kubernetes.default.svc.cluster.local"
```

Comment:
Potentially less useful as a test since the BusyBox implementation of `nslookup` does not retry with an appended search domain during a DNS`NXDOMAIN` error.

We might want to consider moving back to the `gcr.io/kubernetes-e2e-test-images/dnsutils:1.3` image or some other alternative.

Reply:
I think assuming that the test cluster uses the default domain suffix is probably OK?